pages/login.js

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
+import { useRouter } from "next/router";
+
+const Login = () => {
+  const router = useRouter();
+  const [username, setUsername] = useState("");
+  const [password, setPassword] = useState("");
+
+  const handleSubmit = async (ev) => {
+    ev.preventDefault();
+
+    const data = {
+      username,
+      password,
+    };
+
+    try {
+      const res = await fetch("/api/users", {
+        method: "POST",
+        headers: {
+          "Content-Type": "application/json",
+        },
+        body: JSON.stringify(data),
+      });
+      if (res.ok == true) {
+        router.push("/user");
+      } else {
+        alert("Check your login info");
+      }
+    } catch {
+      console.log("Fetch failed", error);
+    }
+  };
+
+  return (
+    <div>
+      <h1>Welcome to login</h1>
```

Comment:
sounds a bit weird "welcome to login" but nothing that will effect the usability.